.github/workflows/release.yml

```diff
@@ -71,7 +71,7 @@ jobs:
         run: go install github.com/goreleaser/goreleaser@latest
 
       - name: Run GoReleaser
-        uses: testifysec/witness-run-action@40aa4ef36fc431a37de7c3faebcb66513c03b934
+        uses: in-toto/witness-run-action@40aa4ef36fc431a37de7c3faebcb66513c03b934
```

Comment:
I think the action has not been moved to @in-toto.

Reply:
https://github.com/in-toto/witness-run-action is a 404.

Reply:
I pushed it without my last amend :|